source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRBaseCardElementRenderer.mm

```diff
@@ -27,8 +27,26 @@ - (UIView *)render:(UIStackView *)viewGroup
             inputs:(NSMutableArray *)inputs
       withCardElem:(std::shared_ptr<BaseCardElement> const &)elem
      andHostConfig:(std::shared_ptr<HostConfig> const&)config
-{ 
+{
    return viewGroup;
 }
 
++ (void)applyLayoutStyle:(NSString *)styleFormat viewsMap:(NSDictionary *)viewsMap
+{
+    NSArray<NSLayoutConstraint *> *constraints;
+
+    for(NSString *key in viewsMap)
+    {
+        NSString *formatString = [[NSString alloc] initWithFormat:styleFormat, key];
+        constraints = [NSLayoutConstraint constraintsWithVisualFormat:formatString
+                                                              options:0
+                                                              metrics:nil
+                                                                views:viewsMap];
+        for(NSLayoutConstraint *constraint in constraints)
+        {
+            constraint.active = YES;
+        }
```

Comment:
A little bit briefer: `[NSLayoutConstraint activateConstraints:constraints];`

Reply:
Thanks

---
In reply to: [143579225](https://github.com/Microsoft/AdaptiveCards/pull/760#discussion_r143579225) [](ancestors = 143579225)